ros/src/computing/perception/prediction/motion_predictor/packages/naive_motion_predict/nodes/naive_motion_predict/naive_motion_predict.cpp

```diff
@@ -16,15 +16,11 @@
 
 #include "naive_motion_predict.h"
 
-NaiveMotionPredict::NaiveMotionPredict() :
-  nh_(),
-  private_nh_("~"),
-  MAX_PREDICTION_SCORE_(1.0)
+NaiveMotionPredict::NaiveMotionPredict() : nh_(), private_nh_("~")
 {
   private_nh_.param<double>("interval_sec", interval_sec_, 0.1);
   private_nh_.param<int>("num_prediction", num_prediction_, 10);
   private_nh_.param<double>("sensor_height_", sensor_height_, 2.0);
-  private_nh_.param<double>("filter_out_close_object_threshold", filter_out_close_object_threshold_, 1.5);
```

Comment:
I feel like this change is a regression rather than an improvement. Are you sure about what you merged here?